api/telemetry/v1alpha1/tenant_types.go

```diff
@@ -64,9 +64,12 @@ type RouteConfig struct {
 // TenantSpec defines the desired state of Tenant
 type TenantSpec struct {
 	SubscriptionNamespaceSelectors []metav1.LabelSelector `json:"subscriptionNamespaceSelectors,omitempty"`
-	LogSourceNamespaceSelectors    []metav1.LabelSelector `json:"logSourceNamespaceSelectors,omitempty"`
-	Transform                      Transform              `json:"transform,omitempty"`
-	RouteConfig                    RouteConfig            `json:"routeConfig,omitempty"`
+
+	// Determines the namespaces from which logs are collected by this tenant.
+	// If initialized with an empty list, logs from all namespaces are collected.
+	LogSourceNamespaceSelectors []metav1.LabelSelector `json:"logSourceNamespaceSelectors,omitempty"`
```

Comment:
Can you please add comments for the other fields as well?
```suggestion
	// Determines the namespaces from which logs are collected by this tenant.
	// If initialized with an empty list, logs from all namespaces are collected.
	// If uninitialized, no logs are collected.
	LogSourceNamespaceSelectors []metav1.LabelSelector `json:"logSourceNamespaceSelectors,omitempty"`
```